elasticapm/contrib/serverless/aws.py

```diff
@@ -400,10 +400,9 @@ def set_metadata_and_context(self, coldstart: bool) -> None:
         metadata["service"]["node"] = {"configured_name": os.environ.get("AWS_LAMBDA_LOG_STREAM_NAME")}
         # This is the one piece of metadata that requires deep merging. We add it manually
         # here to avoid having to deep merge in _transport.add_metadata()
-        if self.client._transport._metadata:
-            node_name = nested_key(self.client._transport._metadata, "service", "node", "name")
-            if node_name:
-                metadata["service"]["node"]["name"] = node_name
```

Comment:
This `metadata.service.node.name` was a separate issue? AFAIK there is no such field in the intake schema.

Reply:
It was an error I found while fixing the metadata bug. I was looking for the wrong field; the code did nothing.